doc/os_requirements.md

```diff
@@ -8,12 +8,25 @@
 
 Edgehog Device Runtime has a number of requirements in order to provide device management features.
 
-## Required Libraries
+## Linux
 
-## Runtime Dependencies
+### Dependencies
+* **Rust** >= 1.58
+* **libsystemd** (optional)
+
+### Runtime Dependencies
 * **[dbus](https://www.freedesktop.org/wiki/Software/dbus/)** (optional): Needed for communicating with 3rd party services, such as RAUC.
 * **[RAUC](https://rauc.io/) ~> v1.5** (optional): Needed for OS updates.
 
-## Filesystem Layout
+### Filesystem Layout
 * **/tmp**: Software updates will be downloaded here.
-* **/data**: Edgehog Device Runtime will store its state here during OTA update process.
\ No newline at end of file
+* **/data**: Edgehog Device Runtime will store its state here during OTA update process.
+
+### Optional features
+* **systemd**: If `edgehog-device-runtime` is a `systemd` service, it can notify `systemd` of its status changes. This is provided via the `rust-systemd` crate, a Rust interface to `libsystemd/libelogind` APIs.
+    
+    To build the `runtime` make sure you have `libsystemd-dev` installed on your system and the systemd feature enabled.
```

Comment:
Why is this indented?

Reply:
Because It is only referred to systemd section, but now I only left indented the `shell cargo command`.